src/cv.jl

```diff
@@ -24,22 +24,24 @@ last valid iteration.
 """
 function cv(
     estimator::LGBMEstimator, X::Matrix{TX}, y::Vector{Ty}, splits;
-    verbosity::Integer = 1, truncate_booster::Bool = true,
+    verbosity::Integer = nothing, truncate_booster::Bool = true,
 ) where {TX<:Real,Ty<:Real}
 
-    ds_parameters = stringifyparams(estimator; verbosity=verbosity)
+    verbosity = isnothing(verbosity) ? estimator.verbosity : verbosity
+    ds_parameters = stringifyparams(estimator)
     full_ds = LGBM_DatasetCreateFromMat(X, ds_parameters)
     LGBM_DatasetSetField(full_ds, "label", y)
 
     return cv(estimator, full_ds, splits, verbosity = verbosity, truncate_booster = truncate_booster)
 end
 
 # Pass Dataset class directly. This will speed up the process if it is part of an iterative process and a pre-constructed dataset is available
-function cv(estimator::LGBMEstimator, dataset::Dataset, splits; verbosity::Integer = 1, truncate_booster::Bool=true)
+function cv(estimator::LGBMEstimator, dataset::Dataset, splits; verbosity::Integer = nothing, truncate_booster::Bool=true)
 
     start_time = now()
     num_data = LGBM_DatasetGetNumData(dataset)
-    parameters = stringifyparams(estimator; verbosity=verbosity)
+    verbosity = isnothing(verbosity) ? estimator.verbosity : verbosity
```

Comment:
These particular changes where functions' verbosity was set to nothing are done for the functions which use stringifyparams to avoid double setting the verbosity (i.e. the default one on estimators + a default one with a specific integer value in the function) which causes a warning. This way the verbosity of the function, i.e. cv, fit! can still be handled via its parameters as before but won't conflict with the now defaulted param on estimator's level.